k8s-bench/tasks/rolling-update-deployment/task.yaml

```diff
@@ -0,0 +1,6 @@
+script:
+- prompt: "Roll out a new image for deployment 'web-app' in namespace 'rollout-test' from 'nginx:1.21' to 'nginx:1.22' ensuring zero downtime"
```

Comment:
How about `update the image of the web-app in rollout-test namespace to 1.22 version. Ensure there is zero downtime (or minimize disruption)` to keep it closer to how a human will say it.

Reply:
I like this eval. A very common use-case.